src/libraries/tests.proj

```diff
@@ -292,8 +293,10 @@
     -->
     <ProjectExclusions Include="$(MSBuildThisFileDirectory)System.Text.RegularExpressions\tests\UnitTests\System.Text.RegularExpressions.Tests.csproj" />
     <ProjectExclusions Include="$(MSBuildThisFileDirectory)System.Text.RegularExpressions\tests\FunctionalTests\System.Text.RegularExpressions.Tests.csproj" />
+    <ProjectExclusions Include="$(MSBuildThisFileDirectory)Microsoft.Extensions.Logging.Abstractions\tests\Microsoft.Extensions.Logging.Generators.Tests\Microsoft.Extensions.Logging.Generators.Roslyn3.11.Tests.csproj" />
```

Comment:
Since it seems like we won't run any generators tests based on discussion from https://github.com/dotnet/runtime/issues/72836, maybe we can add a comment for the related disabled suites to note that they are intended to not run on mobile.
Maybe modifying the commented out L287-293 and remove the `https://github.com/dotnet/runtime/issues/72836`